src/familybot/lib/family_utils.py

```
@@ -52,120 +52,128 @@ def get_family_game_list_url() -> str:
+                    final_price = price_overview.get("final")
+                    if final_price is not None and item[1]:
+                        price_per_person = round(final_price / 100 / len(item[1]), 2)
+                        message_parts.append(
+                            f" which is {price_per_person}$ per person \n"
+                        )
+                    try:
+                        lowest_price = get_lowest_price(int(app_id))
+                        message_parts.append(
+                            f"   The lowest price ever was {lowest_price}$ \n"
+                        )
+                    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)